### DISCUSSION TITLE
Workflow trigger "On issue opened" is now working when issue opened programmatically

### DISCUSSION THREAD

**Discussion by yaroslavbres** [community/community#24953] [Actions]
<p>I am trying to do the scenario:</p>
<ol>
<li>[On Push] to master, First workflow creates an issue (I am using JasonEtco/create-an-issue@v2 action for this)</li>
</ol>
<pre><code class="lang-auto">on:
  push:
    branches:
      - master

jobs:
  create-deploy-issue:
    name: Create Deploy Issue
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Create Deploy Issue
        uses: JasonEtco/create-an-issue@v2
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          DOCKER_TAG: ${{ secrets.DOCKERHUB_IMAGE }}:${{ github.sha }}
</code></pre>
<ol start="2">
<li>[On Issue Opened] The second workflow do some job</li>
</ol>
<pre><code class="lang-auto">on:
  issues: [ opened ]
  
name: Postman tests workflow

jobs:
#some job here
</code></pre>
<p>The problem - Second workflow is not triggered, however, an issue is created</p>
<p>When I manually tries to create the issue, Second workflow triggers succesfully</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The difference is because you’re using the <code>GITHUB_TOKEN</code> to programmatically create the issue, see <a href="https://docs.github.com/en/actions/reference/authentication-in-a-workflow#using-the-github_token-in-a-workflow">Using the <code>GITHUB_TOKEN</code> in a workflow</a>:</p>
<blockquote>
<p>When you use the repository’s <code>GITHUB_TOKEN</code> to perform tasks on behalf of the GitHub Actions app, events triggered by the <code>GITHUB_TOKEN</code> will not create a new workflow run.</p>
</blockquote>
<p>If you want the second workflow to run, you’ll need to use a PAT in the first workflow.</p>

**Comment by yaroslavbres** [comment]
<p>Your suggestion helped me. Thank you!</p>
